### DISCUSSION TITLE
how do I change mitmdump python version?

### DISCUSSION THREAD

**Discussion by behind24proxies** [mitmproxy/mitmproxy#5121] [Q&A]
[windows user]
when I run 
`mitmdump -s file.py`
it gives this error:
`in file.py: no module named keyboard`
in file.py I imported keyboard module like this
`import keyboard`

I already have the package downloaded , to verify I tried making another script [file2.py] containing just one line:
`import keyboard`
and it works completely fine , I ran it using this command:
`python file2.py`

so my conclusion is : mitmdump is using another version of python that is causing issues , so , is there a solution?

**Comment by mhils** [comment]
Our standalone binaries include their own Python environment. You don't want to use those in your case, but install from PyPI instead. 

Details:
https://docs.mitmproxy.org/dev/overview-installation/#installation-from-the-python-package-index-pypi
